CONTRIBUTING.md

```diff
@@ -41,5 +41,4 @@ outlined on that page and do not file a public issue.
 
 ## License
 By contributing to Open Source Education Bot, you agree that
-your contributions will be licensed under the LICENSE file in the root directory
-of this source tree.
\ No newline at end of file
+your contributions will be licensed under the [LICENSE file](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/LICENSE).
```

Comment:
you can simply say under MIT licence

Reply:
and keep the link as it is right?

Reply:
@vj-codes you should use the link as **(./LICENSE)**